packages/react-storage/src/components/FileUploader/ui/FileList/FileDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentClassName, humanFileSize } from '@aws-amplify/ui';
+import { Text, View } from '@aws-amplify/ui-react';
+import { UploadDetailsProps } from './types';
+
+export const UploadDetails = ({
+  displayName,
+  fileSize,
+}: UploadDetailsProps): JSX.Element => {
+  return (
+    <>
+      <View className={ComponentClassName.StorageManagerFileMain}>
```

Comment:
Same comment about classnames being wrong

Reply:
Updated to use name referencing FileUploader